dataservice/api/common/pagination.py

```diff
@@ -112,6 +124,8 @@ def indexd_pagination(q, after, limit):
         next_after = keep[-1].created_at if len(keep) > 0 else after
         # Number of results needed to fulfill the original limit
         remain = limit - len(keep)
+
+        prefetch_indexd()
```

Comment:
It looks like this might not be prefetching the right dids here because `prefetch_indexd` absorbs the value of `after` given to `indexd_pagination` and doesn't track `next_after`